src/applications/MachineShopSimulator.java

```diff
@@ -12,114 +12,12 @@
 
     // data members of MachineShopSimulator
     int timeNow; // current time
-    private int numMachines; // number of machines
-    private int numJobs; // number of jobs
+    int numMachines; // number of machines
+    public int numJobs; // number of jobs

```

Comment:
You generally do not want to be going in this direction and making things more public instead of more private. Is there a reason you've done this here?

Reply:
yes, for know that was just a temporary change, I was planning on making the change to private in a pull-request On another branch. I was mostly just trying to avoid trying to tackle too many items in one PR